mgba-tr.ts

```diff
@@ -0,0 +1,5005 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should be located at `src/platform/qt/ts/mgba-tr.ts`, not `mgba-tr.ts`. Otherwise, looks fine and I'll merge.